src/lib/visuals/visuals.js

```diff
@@ -96,23 +112,39 @@ export default class Visuals {
   setup() {
     this.display.clear();
     this.display.createSvg();
-    const { width } = this.display.dimensions;
-    const albumsHeight = getAlbumHeight(width);
+
+    const backgroundWidth = this.display.dimensions.width;
+    const albumsHeight = getAlbumHeight(backgroundWidth);
+    const backgroundHeight = this.display.dimensions.height - albumsHeight;
 
     const svg = this.display.svg;
 
-    this.mapContainer = createGroup(svg, "map", albumsHeight);
+    this.backgroundContainer =
```

Comment:
Esto no haría falta, no?  Se podría sustituir por:

`this.backgroundContainer = createGroup(svg, "background", albumsHeight)` 

Y utilizar siempre "background" para referirse a ese grupo...